src/components/crossDevice/CrossDeviceSubmit/index.js

```diff
@@ -10,11 +10,16 @@ import style from './style.css'
 import { localised } from '../../../locales'
 
 class CrossDeviceSubmit extends Component {
+  constructor(props) {
+    super(props)
```

Comment:
[Minor] No need to pass props here, it's used only when you want to use props within your consturctor.